### ISSUE TITLE
keyboard accessibility improvements

### ISSUE THREAD

**Issue by trebuan202** [gchq/stroom#5400]
2.1.1 Keyboard (Partially Supports)
Keyboard navigation should flow from top-to-bottom, left-to-right and when you come across a selectable item you should be able to open that item using <enter> or <space bar>.
With keyboard navigation, the hamburger menu ( 3 dots ) which is located in the top-right of the main screen web page, forms part of the top menu of the central panel. When you keyboard navigate across the top menu of the central panel, it does not flow directly to the hamburger menu. Keyboard navigation from the top menu of the central panel moves to the content of the central panel first as opposed to moving to the hamburger menu first. 

With keyboard navigation and selectable items. In the left-hand panel of the main screen, in the case where it includes selectable items ( e.g. Template Pipelines ) you should be able to select or expand this item using a keyboard control like <Enter> or <Space>. You are unable to open such items using a keyboard control.

With keyboard navigation and popup dialog boxes, when these are selected then navigation should subsequently flow within that dialog box. When selecting the “quick syntax filter” help option within the quick filter field in the left-panel, it displays the help text within a popup dialog box, but this dialog box does not get keyboard focus. You are unable therefore to read any of the text within that dialog box.
